packages/safe-core-sdk/README.md

```diff
@@ -496,7 +505,14 @@ Returns a Safe transaction ready to be signed by the owners and executed. The Sa
     refundReceiver, // Optional
     nonce // Optional
   }
-  const safeTransaction = await safeSdk.createTransaction(transactions, options)
+  const safeTransaction = await safeSdk.createTransaction({ safeTransactionData, options })
+  ```
+
+  In addition, the optional `callsOnly` parameter, that is false by default, allows to force the use of the `MultiSendCallOnly` instead of the `MultiSend` contract when sending a batch transaction:
```

Comment:
```suggestion
  In addition, the optional `callsOnly` parameter, that is `false` by default, allows to force the use of the `MultiSendCallOnly` instead of the `MultiSend` contract when sending a batch transaction:
```